cpp/README.md

```diff
@@ -0,0 +1,80 @@
+# Sift C++
+
+This tutorial shows how to build a C++ application that uses the Sift gRPC API.
+
+## Installation
+
+##### Build gRPC and Protocol Buffers
+
+Follow the [official gRPC C++ tutorial](https://grpc.io/docs/languages/cpp/quickstart/#install-grpc) to build and locally install gRPC and Protocol Buffers. You should be able to build and run the [helloworld](https://grpc.io/docs/languages/cpp/quickstart/#build-the-example) example before continuing with the next section.
```

Comment:
Nit, should we include some info here on version compatibility? Like I think we only support protobuf 4 or 5, and I'm assuming there's a corresponding set of grpc versions that are compatible with that as well?